tests/rptest/services/rpk_consumer.py

```diff
@@ -97,8 +100,10 @@ def _consume(self, node):
             if not l:
                 continue
 
-            msg = json.loads(l)
-            self.messages.append(msg)
+            self.message_count += 1
+            if not self._no_save_msgs:
```

Comment:
nit: the double negative can be counterintuitive. Consider changing the boolean flag `_save_msgs` instead.

Reply:
Agreed, I _haven't not_ fixed this (I have fixed this) :-)